CMSgov/mint-app
# [EASI-3465] - Google analytics user property integration

# EASI-3465

## Changes and Description

Did some discovery on the capabilities of GA for react.  Decided on using LD hash for PII safe user id. 
A unique user id will allow us to more accurately report on user metrics

- Added GA user id with LD hash



## How to test this change

Login and on app load verify the user appears in GA (may 24-48 hours) to see the actual id
It is possible to see them flow in real time with debug view, but that requires tag manager integration which was out of scope for the PR (happy to add it if it helps)
You can also inspect you network request to see `uid` properly being passed on the GA network calls

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
